django/contrib/admin/templates/admin/delete_confirmation.html

```diff
@@ -22,12 +22,12 @@
 {% block content %}
 {% if perms_lacking %}
   {% block delete_forbidden %}
-    <p>{% blocktranslate with escaped_object=object %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktranslate %}</p>
+    <p>{% blocktranslate with escaped_object=object %}Deleting the {{ object_name }} "{{ escaped_object }}" would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktranslate %}</p>
     <ul id="deleted-objects">{{ perms_lacking|unordered_list }}</ul>
   {% endblock %}
 {% elif protected %}
   {% block delete_protected %}
-    <p>{% blocktranslate with escaped_object=object %}Deleting the {{ object_name }} '{{ escaped_object }}' would require deleting the following protected related objects:{% endblocktranslate %}</p>
+    <p>{% blocktranslate with escaped_object=object %}Deleting the {{ object_name }} "{{ escaped_object }}" would require deleting the following protected related objects:{% endblocktranslate %}</p>
```

Comment:
If we really want to change this, we should follow typography used in ticket-30400 (check out 42b9a23267f14be39b9b00958e18d5746783208e or 0dafadadb924e09ffd43287604cb4c2c6e353891)
```suggestion
    <p>{% blocktranslate with escaped_object=object %}Deleting the {{ object_name }} “{{ escaped_object }}” would require deleting the following protected related objects:{% endblocktranslate %}</p>
```

Reply:
Thank you. I think I was only considering delete confirmation messages. That seems like a good point.

Reply:
I think similar to 42b9a23267f14be39b9b00958e18d5746783208e, we should update the translations as part of this change. Otherwise we might invalidate translations which are still valid and cause unnecessary work for folks :thinking:

Reply:
Good question, I would assume we'd want to update all of them (maybe @claudep can advise here)

Reply:
I once modified all `.po` files. I asked about this, and claudep gave a reply. (I'm not sure if that reply applies in this case as well.)
https://github.com/django/django/pull/18552#discussion_r1760589990

Reply:
In those cases, we changed the wording, which could change the meaning and so I understand in that case a translator should decide what to do.
But in this case, we are changing punctuation (with no change to the meaning), so I'd like us to double check :slightly_smiling_face:

Reply:
I agree. Understanding these aspects will be helpful for future pr as well. ☺️